docker/nginx/conf.d/server.local.conf

```
@@ -0,0 +1,14 @@
+server {
+    # local server - no not expose this port externally
```

nit: typo. If there is nothing blocking feel free to ignore this, it's not important.
```suggestion
    # local server - do not expose this port externally
```